deploy-manage/autoscaling.md

```diff
@@ -4,67 +4,208 @@ mapped_urls:
+$$$ece-autoscaling-notifications$$$$$$ech-autoscaling-notifications$$$
+::::{tab-set}
+
+:::{tab-item} {{ech}} and {{ecloud}} - Heroku
+In the event that a data tier or machine learning node scales up to its maximum possible size, you’ll receive an email, and a notice also appears on the deployment overview page prompting you to adjust your autoscaling settings to ensure optimal performance.
+:::
+
+:::{tab-item} {{ece}}
+In the event that a data tier or machine learning node scales up to its maximum possible size, a notice appears on the deployment overview page prompting you to adjust your autoscaling settings in order to ensure optimal performance.
+
+A warning is also issued in the ECE `service-constructor` logs with the field `labels.autoscaling_notification_type` and a value of `data-tier-at-limit` (for a fully scaled data tier) or `ml-tier-at-limit` (for a fully scaled machine learning node). The warning is indexed in the `logging-and-metrics` deployment, so you can use that event to [configure an email notification](../explore-analyze/alerts-cases/watcher.md)
+:::
+
+::::
+
+## Restrictions and limitations[ec-autoscaling-restrictions]
+
+$$$ece-autoscaling-restrictions$$$$$$ech-autoscaling-restrictions$$$The following are known limitations and restrictions with autoscaling:
+
+* Autoscaling will not run if the cluster is unhealthy or if the last Elasticsearch plan failed.
+
+::::{tab-set}
+
+:::{tab-item} {{ech}}
+* Trial deployments cannot be configured to autoscale beyond the normal Trial deployment size limits. The maximum size per zone is increased automatically from the Trial limit when you convert to a paid subscription.
+* ELSER deployments do not scale automatically. For more information, refer to [ELSER](../explore-analyze/machine-learning/nlp/ml-nlp-elser.md) and [Trained model autoscaling](../explore-analyze/machine-learning/nlp/ml-nlp-auto-scale.md).
+:::
+
+:::{tab-item} {{ece}}
+* In the event that an override is set for the instance size or disk quota multiplier for an instance by means of the [Instance Overrides API](https://www.elastic.co/docs/api/doc/cloud-enterprise/operation/operation-set-all-instances-settings-overrides), autoscaling will be effectively disabled. It’s recommended to avoid adjusting the instance size or disk quota multiplier for an instance that uses autoscaling, since the setting prevents autoscaling.
+:::
+
+::::
+
+## Enable or disable autoscaling[ec-autoscaling-enable]
+
+$$$ece-autoscaling-enable$$$$$$ech-autoscaling-enable$$$To enable or disable autoscaling on a deployment:
+
+1. **Log in** to the console.
+
+::::{tab-set}
+
+:::{tab-item} {{ech}}
+[Elasticsearch Service Console](https://cloud.elastic.co?page=docs&placement=docs-body)
+:::
+
+:::{tab-item} {{ece}}
+[Cloud UI](../deploy-manage/deploy/cloud-enterprise/log-into-cloud-ui.md)
+:::
+
+:::{tab-item} {{ecloud}} -  Heroku
+[Elasticsearch Add-On for Heroku console](https://cloud.elastic.co?page=docs&placement=docs-body)
+:::
+::::
```

Comment:
we need to land on a generic term, but let's just squish them together for now. no need to provide a heroku option.

we no longer use the term "Elasticsearch Service" - only "Elastic Cloud" or "Elastic Cloud Hosted" depending on whether you're referring to the organization or the deployment.

```suggestion
1. Log in to the ECE [Cloud UI]((/deploy-manage/deploy/cloud-enterprise/log-into-cloud-ui.md) or  [{{ecloud}} Console](https://cloud.elastic.co?page=docs&placement=docs-body).
```